### PR TITLE
stars

### PR DESCRIPTION
**Pull request by LBondarchuk**
- [DEMO LINK](https://LBondarchuk.github.io/layout_stars/)
- [TEST REPORT LINK](https://LBondarchuk.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
```
Comment (agniya-i) [inline src/style.css:38]: combine selectors to not repeat code

**Review by agniya-i — CHANGES_REQUESTED** [review]
Please try to fix failed tests

File: src/style.css (hunk 1 above)
Comment (LBondarchuk) [reply src/style.css:38]: I don't understand how to do it yet. Given the conditions.

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- You have a lot of failed tests. Try to fix them. Check padding, maybe. If you have questions, ask them in the fe_chat
image

https://user-images.githubusercontent.com/99403198/216955643-268b62cb-b7b9-4726-994b-524a22f96195.mov

**Pull request closed without merging**